crates/app/ironclaw_cli/.skills/portfolio/scripts/backtest_strategy.py

```
@@ -0,0 +1,79 @@
+def load_config():
+    raw = tool_invoke("memory_read", {
+        "target": f"projects/{PROJECT_ID}/config.json",
+    })
+    return json.loads(raw["content"] if isinstance(raw, dict) else raw)
+
+def backtest(strategy_doc):
+    snaps = load_history()
+    config = load_config()
+    results = []
+    for snap in snaps:
+        propose = tool_invoke("portfolio", {
```

Ruff (0.16.1)

[error] Undefined name `tool_invoke`

(F821)